models/br_ibge_censo_2022/schema.yml

```diff
@@ -836,3 +836,293 @@ models:
+              field: id_municipio
+      - name: tipo_esgotamento_sanitario
+        description: Tipo de esgotamento sanitário
+      - name: domicilios
+        description: Domicílios particulares permanentes ocupados
+  - name: br_ibge_censo_2022__domicilio_canalizacao_abastecimento_agua_municipio
+    description: Domicílios particulares permanentes ocupados, por existência de canalização de água e principal forma de abastecimento de água
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - ano
+            - id_municipio
+            - tipo_abastecimento_agua
+            - tipo_canalizacao_agua
+      - not_null_proportion_multiple_columns:
+          at_least: 0.05
+    columns:
+      - name: ano
+        description: Ano
+      - name: id_municipio
+        description: ID Município IBGE 7 dígitos
+        tests:
+          - relationships:
+              to: ref('br_bd_diretorios_brasil__municipio')
+              field: id_municipio
+      - name: tipo_abastecimento_agua
+        description: Principal forma de abastecimento de água
+      - name: tipo_canalizacao_agua
+        description: Existência de canalização de água
+      - name: domicilios
+        description: Domicílios particulares permanentes ocupados
+  - name: br_ibge_censo_2022__domicilio_ligacao_abastecimento_agua_municipio
+    description: Domicílios particulares permanentes ocupados, por existência de ligação à rede geral de distribuição de água e principal forma de abastecimento de água.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns: [ano, id_municipio, tipo_ligacao_rede_geral]
+      - not_null_proportion_multiple_columns:
+          at_least: 0.05
+    columns:
+      - name: ano
+        description: Ano
+        tests:
+          - relationships:
+              to: ref('br_bd_diretorios_data_tempo__ano')
+              field: ano
+      - name: id_municipio
+        description: ID Município IBGE 7 dígitos
+        tests:
+          - relationships:
+              to: ref('br_bd_diretorios_brasil__municipio')
+              field: id_municipio
+      - name: tipo_ligacao_rede_geral
+        description: Existência de ligação à rede geral de distribuição de água e
+          principal forma de abastecimento de água
+      - name: domicilios
+        description: Domicílios particulares permanentes ocupados
+  - name: br_ibge_censo_2022__domicilio_tipo_municipio
+    description: Domicílios particulares permanentes ocupados, por tipo de domicílio
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns: [ano, id_municipio, tipo_domicilio]
+      - not_null_proportion_multiple_columns:
+          at_least: 0.05
+    columns:
+      - name: ano
+        description: Ano
+        tests:
+          - relationships:
+              to: ref('br_bd_diretorios_data_tempo__ano')
```

Comment:
A relationships referente a ano, ficaria:

          - relationships:
              to: ref('br_bd_diretorios_data_tempo__ano')
              field: ref('ano.ano')